validator-generator/src/main/java/io/avaje/validation/generator/ElementAnnotationContainer.java

```diff
@@ -49,6 +50,7 @@ static ElementAnnotationContainer create(Element element) {
 
     final var annotations =
         element.getAnnotationMirrors().stream()
+            .filter(m -> !hasValid && !ValidPrism.isInstance(m))
```

Comment:
Oh, this might be the issue ... 

Currently the `@Valid` annotation is actually used like below in the generated `AShipValidationAdapter`  ...

```java
    this.crewValidationAdapter = 
        ctx.<List<ACrew>>adapter(Valid.class, Map.of())
            .list()
            .andThenMulti(ctx.adapter(ACrew.class));
```

So it that is filtered out then this will struggle here I think.